python/cuml/cuml/cluster/hdbscan/hdbscan.pyx

```diff
@@ -370,7 +370,7 @@ cdef class _HDBSCANState:
             cluster_persistence = CumlArray(
                 data=_cupy_array_from_ptr(
                     <size_t>self.hdbscan_output.get_stabilities(),
-                    (1, self.n_clusters),
+                    (self.n_clusters,),
```

Comment:
Before this had the incorrect shape. It now matches our documented shape, as well as the shape used in `hdbscan.HDBSCAN`. I view this as a bugfix.

Reply:
Agreed, this is a bug fix.